### PR TITLE
fix(wc_yotpo.php): used yotpo_get_arr_value in wc_yotpo.php to eliminate warnings

### PR DESCRIPTION
**Pull request by p-bartosz**

### CODE REVIEW

File: trunk/lib/utils/general-usage-functions.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?php
 
-function yotpo_get_arr_value(array $object = [], string $key, $default = '') {
+function yotpo_get_arr_value(array $object = [], string $key = '', $default = '') {
```
Comment (adam-wieckowski-yotpo) [inline trunk/lib/utils/general-usage-functions.php:3]: what's the reason for default value? is this function used anywhere without key? if so - why?

File: trunk/wc_yotpo.php [hunk 2]
```diff
@@ -233,8 +234,9 @@ function generate_star_ratings_widget_code() {
 	global $product;
 	$yotpo_settings = get_option('yotpo_settings',wc_yotpo_get_default_settings());
 	$use_v3_widgets = use_v3_widgets();
-	$reviews_widget_id = $yotpo_settings['v3_widgets_ids']['reviews_widget'];
-	$star_ratings_widget_id = $yotpo_settings['v3_widgets_ids']['star_rating'];
+	$v3_widgets_ids = yotpo_get_arr_value($yotpo_settings, 'v3_widgets_ids', []);
```
Comment (adam-wieckowski-yotpo) [inline trunk/wc_yotpo.php:237]: is it used anywhere?

File: trunk/lib/utils/general-usage-functions.php (hunk 1 above)
Comment (p-bartosz) [reply trunk/lib/utils/general-usage-functions.php:3]: image

File: trunk/wc_yotpo.php (hunk 2 above)
Comment (p-bartosz) [reply trunk/wc_yotpo.php:237]: two lines below

File: trunk/lib/utils/general-usage-functions.php (hunk 1 above)
Comment (adam-wieckowski-yotpo) [reply trunk/lib/utils/general-usage-functions.php:3]: yes, I saw that but deprecation was in PHP8 so why this error showed up now? unless someone updated php

**Review by adam-wieckowski-yotpo — APPROVED** [review]

**Pull request merged**